<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>pqAddSourceDialog</class>
 <widget class="QDialog" name="pqAddSourceDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Select Source</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="2" column="1" colspan="2" >
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="0" column="2" >
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>16</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0" colspan="2" >
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1" >
      <widget class="QLineEdit" name="SourceName" >
       <property name="whatsThis" >
        <string>Used to enter the desired source name.</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2" >
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>16</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="0" >
      <widget class="QLabel" name="SourceLabel" >
       <property name="text" >
        <string>Source</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3" >
      <widget class="QPushButton" name="CancelButton" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3" >
      <widget class="QPushButton" name="OkButton" >
       <property name="text" >
        <string>OK</string>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1" >
    <widget class="QComboBox" name="SourceGroup" >
     <property name="whatsThis" >
      <string>Lists the source groups.</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>0</number>
     </property>
     <item>
      <widget class="QToolButton" name="BackButton" >
       <property name="toolTip" >
        <string>Back</string>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="../pqComponents.qrc" >:/pqWidgets/Icons/pqBack16.png</iconset>
       </property>
       <property name="autoRaise" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="UpButton" >
       <property name="toolTip" >
        <string>Up One Level</string>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="../pqComponents.qrc" >:/pqWidgets/Icons/pqFolderUp16.png</iconset>
       </property>
       <property name="autoRaise" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="NewFolderButton" >
       <property name="toolTip" >
        <string>Create New Group</string>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="../pqComponents.qrc" >:/pqWidgets/Icons/pqFolderNew16.png</iconset>
       </property>
       <property name="autoRaise" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="AddToFavoritesButton" >
       <property name="toolTip" >
        <string>Add To Favorites</string>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="../pqComponents.qrc" >:/pqWidgets/Icons/pqAddFavorites16.png</iconset>
       </property>
       <property name="autoRaise" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" >
    <widget class="QLabel" name="SourceGroupLabel" >
     <property name="text" >
      <string>Source Group</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2" >
    <widget class="QListView" name="SourceList" >
     <property name="whatsThis" >
      <string>Displays the sources in the currently selected group.</string>
     </property>
     <property name="isWrapping" stdset="0" >
      <bool>true</bool>
     </property>
     <property name="resizeMode" >
      <enum>QListView::Adjust</enum>
     </property>
    </widget>
   </item>
   <item rowspan="2" row="1" column="0" >
    <widget class="QListView" name="SourceHistory" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>0</hsizetype>
       <vsizetype>7</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize" >
      <size>
       <width>140</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="whatsThis" >
      <string>Displays the most recently selected sources.</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <tabstops>
  <tabstop>SourceName</tabstop>
  <tabstop>OkButton</tabstop>
  <tabstop>CancelButton</tabstop>
  <tabstop>SourceList</tabstop>
  <tabstop>SourceGroup</tabstop>
  <tabstop>BackButton</tabstop>
  <tabstop>UpButton</tabstop>
  <tabstop>NewFolderButton</tabstop>
  <tabstop>AddToFavoritesButton</tabstop>
  <tabstop>SourceHistory</tabstop>
 </tabstops>
 <resources>
  <include location="../pqComponents.qrc" />
 </resources>
 <connections/>
</ui>
